hawkbit-repository/hawkbit-repository-test/src/main/java/org/eclipse/hawkbit/repository/test/util/TestdataFactory.java

```diff
@@ -346,6 +346,26 @@ public DistributionSet createDistributionSet(final String prefix, final String v
     }
 
     /**
+     * Create a list of {@link DistributionSet}s without modules, i.e.
+     * incomplete.
+     * 
+     * @param number
+     *            of {@link DistributionSet}s to create
+     * @return {@link List} of {@link DistributionSet} entities
+     */
+    public List<DistributionSet> createDistributionSetsWithoutModules(final int number) {
+
+        final List<DistributionSet> sets = new ArrayList<>();
```

Comment:
That does not help much. Lists.newArrayListWithExpectedSize(number) is the right one. But you are generally right of course.